.github/workflows/ci-linux-on-main-pr.yml

```diff
@@ -26,14 +26,13 @@ jobs:
         run: |
           sudo apt-get update -y
           sudo apt-get install -y 
-          sudo apt-get remove -y llvm-12
+          sudo apt-get remove -y llvm-12 llvm-11 clang-format-11 clang-tidy-11 
           sudo apt-get install -y curl pkg-config findutils  wget
-          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11 
-          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime opt 
+          sudo apt install -y clang-13 cmake clang-format-13 clang-tidy-13 
+          sudo apt-get install -y llvm-13 lldb-13 llvm-13-dev libllvm11 llvm-13-runtime opt 
           sudo apt-get install -y nodejs npm
           sudo apt install -y python3 python3-pip 
-          sudo apt-get remove -y llvm-10
-          sudo ln -s /usr/lib/llvm-11/bin/opt /usr/bin/opt
+          sudo ln -s /usr/lib/llvm-13/bin/opt /usr/bin/opt
```

Comment:
Up to you.
It probably makes sense to create a constant (variable) that contains the version of active LLVM (in this case `13`), and use that constant everywhere. Otherwise there is a chance that some time later someone will be updating version 13 to 14+, will update the green lines 31 - 32, but will forget to update `13` in the green line 35.

Reply:
I agree that it could make sense to use a environment variable if upgrades would happen often. The downside is that it would lower readability and I think that given that we do not expect to upgrade LLVM more than once a year at most, I would prefer the to keep it the way it is.